manifests/FTB/FTB/1.0.8.yaml

```diff
@@ -0,0 +1,15 @@
+Id: FTB.FTB
+Version: 1.0.8
+Name: FTBApp
+Publisher: Feed The Beast
+License: LGPL 2.1 (frontend), CC BY-NC-ND (backend)
+LicenseUrl: https://github.com/FTBTeam/FTB-App/blob/master/LICENSE.md
+AppMoniker: ftb
+Tags: game, minecraft, ftb
+Homepage: https://www.feed-the-beast.com/
+Installers:
+  - Arch: x64
+    Url: https://apps.modpacks.ch/FTBApp/release/1.0.8/FTBA_windows-x64_1_0_8.exe
+    Sha256: 79372BA75C585D1F4DBE288C71591735FD288E2C821CF94940A8CC2551C04948
+    InstallerType: exe
```

Comment:
Do we know what installer they use? nullsoft, etc

Reply:
After reading install4j's docs, I figured out that it's `-q`.